modules/ocf_irc/manifests/services.pp

```diff
@@ -7,7 +7,7 @@
   }
 
   $irc_creds = lookup('irc_creds')
-  $root_nicks = ['waf', 'nattofriends', 'ckuehl', 'jvperrin', 'mattmcal', 'abizer', 'dkessler', 'cooperc', 'jaw']
+  $root_nicks = ['waf', 'nattofriends', 'ckuehl', 'jvperrin', 'mattmcal', 'abizer', 'dkessler', 'cooperc', 'jaw', 'kmo', 'ronitnath', 'njha', 'fydai']
```

Comment:
This nick isn't registered

Reply:
oh hm we have to be registered in irc?

Reply:
yes these all have to be nickserv registered names, they aren't related to OCF accounts